src/com/dotcms/rest/api/v1/system/AppConfigurationHelper.java

```diff
@@ -35,8 +76,8 @@
 	 * @return The navigation menu items.
 	 */
 	public Object getMenuData(final HttpServletRequest request) {
-		final MenuResource menuResource = new MenuResource();
-		Object entity = new Object();
+
+		Object entity = null;
 		try {
 			final Response menuResponse = menuResource.getMenus(MenuResource.App.CORE_WEB.name(), request);
```

Comment:
Since we're updating this class, it'd be great to change this old code: Instead of going through the resource class, use the Helper class for the MenuResource.